### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *bucolic-parfait-7460f9* failed.

### ISSUE THREAD

**Issue by Dargon789** [Dargon789/hardhat-project#1744]
### <span aria-hidden="true">❌</span> Deploy Preview for *bucolic-parfait-7460f9* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | da2b310773a9998a35bcdf6304c4a821d117714a |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/bucolic-parfait-7460f9/deploys/665c92d1d7717100088a74b4 |

_Originally posted by @netlify[bot] in https://github.com/Dargon789/hardhat-project/issues/6#issuecomment-2143906374_
